commit 67aa8be4cb43cd94bc322fed8bdba48b3c8719c4
Author: Andreas Krebbel <krebbel@linux.vnet.ibm.com>
Date:   Mon May 29 12:38:11 2017 +0200

    S/390: Add missing operand to tb instruction
    
    gas/ChangeLog:
    
    2017-05-30  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
    
            * testsuite/gas/s390/esa-g5.d: Add missing operand to tb
            instruction.
            * testsuite/gas/s390/esa-g5.s: Likewise.
    
    opcodes/ChangeLog:
    
    2017-05-30  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
    
            * s390-opc.txt: Add missing operand to tb instruction.

--- a/opcodes/s390-opc.txt
+++ b/opcodes/s390-opc.txt
@@ -228,7 +228,7 @@ b246 stura RRE_RR "store using real address" g5 esa,zarch
 2f swr RR_FF "subtract unnormalized (long)" g5 esa,zarch
 37 sxr RR_FEFE "subtract normalized (ext.)" g5 esa,zarch
 b24c tar RRE_AR "test access" g5 esa,zarch
-b22c tb RRE_0R "test block" g5 esa,zarch
+b22c tb RRE_RR "test block" g5 esa,zarch
 91 tm SI_URD "test under mask" g5 esa,zarch
 b236 tpi S_RD "test pending interruption" g5 esa,zarch
 e501 tprot SSE_RDRD "test protection" g5 esa,zarch
